README.md

```diff
@@ -1,6 +1,6 @@
 # OSIsoft Cloud Services Python Library Sample
 
-**Version:** 0.1.18_preview
+**Version:** 0.2.0_preview
 
 [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/osisoft.sample-ocs-sample_libraries-python?branchName=main)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2622&branchName=main)
```

Comment:
Not developed against 3.7...  Also might need a note about requiring 3.9

Reply:
Added some info and cleaned up the README a bit.